llvm/tools/llvm-exegesis/lib/SnippetFile.cpp

```diff
@@ -131,6 +131,12 @@ class BenchmarkCodeStreamer : public MCStreamer, public AsmCommentConsumer {
       Result->Key.MemoryMappings.push_back(std::move(MemMap));
       return;
     }
+    if (CommentText.consume_front("SNIPPET-ADDRESS")) {
+      // LLVM-EXEGESIS-SNIPPET-ADDRESS <address>
+      Result->Key.SnippetAddress =
+          std::stol(CommentText.trim().str(), nullptr, 16);
```

Comment:
Please use `llvm::to_integer` so that we can diagnose a bad value.

Reply:
Thanks for the suggestion! Added error handling in the latest revision.